inference-engine/src/gna_plugin/gna_plugin_config.cpp

```diff
@@ -71,7 +71,7 @@ void Config::UpdateFromMap(const std::map<std::string, std::string>& config) {
                 key.erase(0, 1);
                 try {
                     input_index = std::stoi(key);
-                    if (input_index < 0 | input_index > 99) {
+                    if (input_index > 99) {
```

Comment:
might be add informative error?

Reply:
We have it! See the catch section. I only removed a redundant check (an unsigned int cannot be less than 0).